Misc/NEWS.d/next/Library/2024-10-10-18-33-31.gh-issue-125235.0kOB5I.rst

```diff
@@ -0,0 +1,2 @@
+Keep :mod:`_tkinter` TCL paths in venv pointing to base installation on
```

Comment:
Maybe you should use it's upper layer encapsulation instead of the original private module.

Reply:
Thanks, resolved.